sources/packages/backend/apps/db-migrations/src/sql/SINValidations/Rollback-alter-column-gender-sent.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE
+  sims.sin_validations
+SET
+  gender_sent = NULL
+WHERE
+  LENGTH(gender_sent) > 10;
```

Comment:
Is it desired for rollback ? IMO, we must just do the alter statement.

If at all there is any need to rollback, we can handle it manually. 

@andrewsignori-aot  ?

Reply:
I would agree with @dheepak-aot. No values greater than 10 will be expected in any DB since they are not supported. The rollback script, if executed, should be executed at release time.